Chap_API_Data_Mgmt.tex

```diff
@@ -243,11 +245,12 @@ \subsection{\code{PMIx_Data_unpack}}
 
 
 \begin{arglist}
```

Comment:
"the description" -> the \refstruct{pmix_proc_t}

Reply:
Not changing this as described above (which would result in "Pointer to a \refstruct{pmix_proc_t} structure containing the \refstruct{pmix_prot_t}").  We think this double occurence of pmix_proc_t is still confusing so went with "Pointer to a \refstruct{pmix_proc_t} structure of the process that packed the provided buffer."